src/org/opendatakit/briefcase/reused/http/response/ServerError.java

```diff
@@ -16,17 +16,36 @@
 
 package org.opendatakit.briefcase.reused.http.response;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.opendatakit.briefcase.reused.http.response.Response.uncheckedReadContent;
+
+import java.io.ByteArrayInputStream;
+import java.util.Optional;
 import java.util.function.Function;
 import java.util.function.Supplier;
+import org.apache.http.HttpResponse;
 import org.opendatakit.briefcase.reused.http.HttpException;
+import org.opendatakit.briefcase.reused.http.Request;
 
 class ServerError<T> implements Response<T> {
   private final int statusCode;
   private final String statusPhrase;
+  private final String serverErrorResponse;
 
-  ServerError(int statusCode, String statusPhrase) {
+  private ServerError(int statusCode, String statusPhrase, String serverErrorResponse) {
     this.statusCode = statusCode;
     this.statusPhrase = statusPhrase;
+    this.serverErrorResponse = serverErrorResponse;
+  }
+
+  static <U> ServerError<U> from(Request<U> request, HttpResponse response) {
```

Comment:
`request` is unused?

Reply:
That one is there just for conformity with other factories in sibling subtypes of Response, but we could remove it if you think it's confusing.

Reply:
Perhaps remove the confusion with a comment, or a warning-disabling statement?